test

```diff
@@ -391,7 +391,7 @@ function shellcheck_pass {
 
 function markdown_you_pass {
 	# eschew you
-	yous=$(find . -name \*.md ! -path './vendor/*' ! -path './Documentation/v2/*' ! -path './gopath.proto/*' -exec grep -E --color "[Yy]ou[r]?[ '.,;]" {} + || true)
+	yous=$(find . -name \*.md ! -path './vendor/*' ! -path './Documentation/*' ! -path './gopath.proto/*' -exec grep -E --color "[Yy]ou[r]?[ '.,;]" {} + || true)
```

Comment:
You know doc better and as you mentioned in the past discussion we should make this change, so removing the restriction on using 'you' is totally fine to remove with me, but copying @gyuho @xiang90 @philips if they have any comment here. Thanks!